schema/results.sql

```diff
@@ -17,10 +17,12 @@ CREATE TABLE results (
 	id varchar(64),
 
 	name varchar(64),
+	annotations BLOB,
+
 	created_time timestamp default current_timestamp not null,
 	updated_time timestamp default current_timestamp not null,
 	
-	annotations BLOB,
+	etag text,
```

Comment:
IIUC, using text limits our ability to include this in other places in the schema (notably indexes). Can we guarantee a maximum size here? If so, we might want to consider moving to varchar.

Reply:
Yeah, It should be 128, and done.